<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_scan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWid">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>1111</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
border:none

}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Scan</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Scan</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_camera">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>821</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="clockOutLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="close_cam">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Close Camera</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_clock_status">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Records</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>491</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Admin  Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Panel</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Admin  Panel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_upload">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>430</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>481</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="u_name">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_face">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Add Face</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>-10</y>
      <width>891</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Face Recognition System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_admin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_record">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Records</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/mini.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/close.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
